app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -39,6 +39,7 @@ def error_hash(message)
     def extract_pii_from_doc(user, store_in_session: false)
       if defined?(idv_session) # hybrid mobile does not have idv_session
```

Comment:
I definitely forgot about the hybrid flow, but I _think_ this will work, since `LinkSentController` includes `DocumentCaptureConcern` and calls `extract_pii_from_doc`. However, I think we should have spec coverage for that.

Reply:
Good point. Working on that now.

Reply:
Added! ([diff](https://github.com/18F/identity-idp/pull/11742/files/e6b40ea66061ef16b6f4bc0684fe538053e16fc8))